src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,133 @@
-import { PeopleFilters } from './PeopleFilters';
+import { useEffect, useState } from 'react';
+import { useMatch, useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
 import { Loader } from './Loader';
+import { PeopleFilters } from './PeopleFilters';
 import { PeopleTable } from './PeopleTable';
+import { ErrorTypes, Person } from '../types';
+import { getCentry, sortedPeople } from '../utils';
 
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isPeopleLoaded, setIsPeopleLoaded] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isErrorNoteShown, setErrorNoteShown] = useState(false);
+  const match = useMatch('/people/:slug');
+  const selectedPerson = match?.params.slug;
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  useEffect(() => {
+    async function fetchPerson() {
+      setIsLoading(true);
+      setIsPeopleLoaded(false);
+      setErrorNoteShown(false);
+
+      try {
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+      } catch {
+        setErrorNoteShown(true);
+      } finally {
+        setIsLoading(false);
+        setIsPeopleLoaded(true);
+      }
+    }
+
+    fetchPerson();
+  }, []);
+
+  let visiblePeople;
```

Comment:
```suggestion
const visiblePeople = order ? getPeopleShow().reverse() : getPeopleShow();
```
create after getPeopleShow